### DISCUSSION TITLE
Beta Project UI - "It's a trap!" (that's a StarWars quote) - but I really like the Beta Projects

### DISCUSSION THREAD

**Discussion by Jon1776** [community/community#8782] [Projects and Issues]
I really like the new beta projects, but I would greatly appreciate having a "projects" button on the top of the screen so I can navigate to my other projects.  

A beta project doesn't have any useful navigation buttons/links on the top of the page making it hard to get back to other projects or places inside GitHub.  

I hope new nav options because Beta Projects looks really great

Here is what the traditional nav bar looks like in projects
<img width="1442" alt="Screen Shot 2021-12-13 at 6 07 01 AM" src="https://user-images.githubusercontent.com/38586819/145801707-d95e669d-e88d-4ca9-bc0f-40eda93930e8.png">
Here is the Beta Project (with no nav bars) 
<img width="1442" alt="Screen Shot 2021-12-13 at 6 08 37 AM" src="https://user-images.githubusercontent.com/38586819/145801898-ca33ce2e-d0ba-4376-96a0-356ad571f38c.png">

**Comment by Katsute** [comment]
https://github.com/github/feedback/discussions/6560

**Comment by mkungla** [comment]
Collected all related discussions here: https://github.com/github/feedback/discussions/10448
